apis/supernova/src/hooks.js

```diff
@@ -1064,14 +1064,28 @@ export function useRenderState() {
   };
 }
 
+/**
+ * Blur function
+ * @name BlurFunction
+ * @function
+ * @param {boolean} blurState parameter to indicate blur state
+ */
+
+/**
+ * Focus Selection function
+ * @name FocusSelectionFunction
+ * @function
+ * @param {boolean} focusState parameter to indicate focus state
```

Comment:
this bool determines if the selection toolbar should be focuses the first or last element in the toolbar. if true, the last item is focused

Reply:
it is called `focusLast` in the cope. I think it can be optional as well

Reply:
not sure if that can be optional because there is no check before using it in here: https://github.com/qlik-oss/nebula.js/blob/64a3b201dd9938a94765a4948230b3a8e7337624/apis/supernova/src/hooks.js#L1131

Reply:
it will be undefined, which will act falsey later where it is used

Reply:
you could add  `= false` on that line for clarity